changelog/7336.bugfix.rst

```diff
@@ -0,0 +1 @@
+The pytest ``--log-file`` cli or ``log_file`` ini marker is now relative to the configs ``inifile`` directory.
```

Comment:
this is a breaking change, the filename shoudld reflect that

Reply:
Hmm good point, although the docs say otherwise, indeed this might break users.